src/Application.php

```diff
@@ -128,8 +128,10 @@ public function services(ContainerInterface $container): void
     protected function bootstrapCli(): void
     {
         try {
+            $this->addPlugin('Cake/Repl');
             $this->addPlugin('Bake');
         } catch (MissingPluginException $e) {
+            debug($e->getMessage());
```

Comment:
Including `config/bootstrap` file as done [here](https://github.com/cakephp/cakephp/blob/4ac765fdd47d6b185bf9c55dfd5bb068b3daba2c/phpstan.neon#L10) should make phpstan happy.

Reply:
Whooops this wasn't supposed to be here.